content/en/docs/controller_flow.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Control flow
+---
+
+The following image illustrate the interactions between the Build, BuildRun controller and the Tekton Pipeline controller:
```

Comment:
This is a bit more of an architecture article, and not meant for end users. For now I say it can live in shipwright-io/build